eZ/Publish/Core/MVC/Symfony/SiteAccess/Matcher/HostElement.php

```diff
@@ -37,10 +37,15 @@ class HostElement implements VersatileMatcher
     /**
      * Constructor.
      *
-     * @param int $elementNumber Number of elements to take into account.
+     * @param int|array $elementNumber Number of elements to take into account.
      */
     public function __construct($elementNumber)
     {
+        if (is_array($elementNumber) && isset($elementNumber['value'])) {
```

Comment:
@andrerom: The reasoning for keeping int as well as array was exactly for BC purposes, but either solution would be fine by me:
 - use both int|array, optionally add additional tests. 
 - use only array, add BC note, fix/update tests.
 - modify configuration parser somehow (hackish IMHO).